app/views/terms-and-conditions/private-beta-3.njk

```diff
@@ -48,7 +48,7 @@
                 <li>(c) you are the person responsible for a Single Species on a permanent or temporary basis on the date or dates the Vet carries out the Review;</li>
                 <li>(d) you will have the relevant minimum number of the Single Species as described in condition 3.3 on the date or dates the Vet carries out the Review; </li>
                 <li>(e) You will have the relevant minimum number of Single Species as described in condition 4.1 on the date or dates the Vet carries out the testing for the Review. </li>
-                <li>(f) have not applied for and accepted an agreement offer for a Single Species Review from another AWHR beta/phase in the last 10 months  </li>
+                <li>(f) You have not applied for and accepted an agreement offer for a Single Species Review from another AWHR beta/phase in the last 10 months, unless the 10 months has been waived in line with your previous agreements’ terms and conditions;</li>
```

Comment:
Your change is fine but I've noticed that some of the list items begin with 'you' and others 'You'. Seems a bit strange to have this inconsistency in content

Reply:
Cheers, raised this question will content and prototype team, will address anything they feed back separately